parse.go

```diff
@@ -109,7 +109,8 @@ type Parser struct {
 //
 // frameChannel is an optional channel (can be nil) upon which DICOM image frames will be sent as they are parsed (if
 // provided).
-func NewParser(in io.Reader, bytesToRead int64, frameChannel chan *frame.Frame) (*Parser, error) {
+func NewParser(in io.Reader, bytesToRead int64, frameChannel chan *frame.Frame, opts ...ParseOption) (*Parser, error) {
```

Comment:
we should probably have a test w/ the new API where the skipHeaderRead option is provided and we can confirm that the header was not read or that things work as expected--would you be able to help add something to that effect?